Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -275,19 +275,25 @@ def get_hosts_scan_info(self, params: Optional[dict] = None) -> List[dict]:
         """
         return self._http_request(method="GET", url_suffix="/hosts", params=params)
 
-    def get_impacted_resources(self, cve: str) -> dict:
+    def get_impacted_resources(self, cve: str, resource_type: str) -> dict:
         """
         Get the impacted resources that are based on a specific CVE.
 
         Args:
             cve (str): The CVE from which impacted resources will be retrieved.
+            resource_type (str): ResourceType is the single resource type to return vulnerability data for.
 
         Returns:
             dict: the impacted resources from the CVE.
         """
+        if resource_type:
+            return self._http_request(
+                method="GET", url_suffix="/stats/vulnerabilities/impacted-resources",
+                params={"cve": cve, "resourceType": resource_type}
+            )
         return self._http_request(
-            method="GET", url_suffix="/stats/vulnerabilities/impacted-resources", params={"cve": cve}
-        )
+            method="GET", url_suffix="/stats/vulnerabilities/impacted-resources",
+            params={"cve": cve})
```

Comment:
please add comment to what it returns by default if not specifying the resource type query parameter